internal/backends/gl/glcontext.rs

```diff
@@ -60,8 +70,15 @@ impl OpenGLContext {
         }
     }
 
+    pub fn is_current(&self) -> bool {
+        #[cfg(not(target_arch = "wasm32"))]
+        return matches!(self.0.borrow().as_ref().unwrap(), OpenGLContextState::Current(_));
+        #[cfg(target_arch = "wasm32")]
+        return matches!(self.0.borrow().as_ref().unwrap(), OpenGLContextState::Current { .. });
```

Comment:
```suggestion
        return matches!(self.0.borrow().as_ref().unwrap(), OpenGLContextState::Current { .. });
```

Reply:
Awesome, I didn't know that `{ .. }` also matches a tuple. Then I can get rid of the function altogether :)